cfpb/consumerfinance.gov
# fix(csp): add exceptions for ANDI A11y Tool

We helped the ITAP team over in RegTech make some exceptions to our CSP during testing so they could use the ANDI accessibility tool for their accessibility audits, and they were hoping they could also run it on cf.gov. This PR adds the few CSP tweaks they need for the tool to start working.

You can see the current CSP issues by following the "How to test this PR" steps or take a peek at the screenshots.

Closes #8541 

## Notes and todos

- Before this comes out of draft I need to...
  - ~~Test this out in a dev environment~~
  - ~~Have someone from ITAP confirm that it's working as expected~~
  - ~~Talk to someone from D+D about [the Akamai shenanigans that need to be done](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/cfgov/settings/base.py#L469-L480) before this change could go live~~
  - ~~Checked with Wyatt about the CSP~~

## Additions
- Add exceptions for ANDI A11y Tool to the CSP
  - ANDI scripts and the jquery version it uses
  - ANDI css styles
  - ANDI images for its icons


## How to test this PR

1. Install the ANDI accessibility tool bookmarklet in your browser [using these instructions](https://www.ssa.gov/accessibility/andi/help/install.html)
2. Click the ANDI button when on consumerfinance.gov (currently on DEV2)
3. Does the tool appear and work as expected?


## Screenshots

__Before__
<img width="1361" alt="Screenshot 2024-08-22 at 4 56 16 AM" src="https://github.com/user-attachments/assets/b966cf87-a700-4e28-ba59-ae7a2d3198a7">


__After__

<img width="991" alt="Screenshot 2024-08-22 at 4 56 54 AM" src="https://github.com/user-attachments/assets/cd9435c8-1940-4a34-88ea-475c38c96963">
